### PR TITLE
Update passwordvault.md

### PR DESCRIPTION
**Pull request by ptorr-msft**
Updated description to better reflect behaviour of non-UWP apps.

### CODE REVIEW

**Comment by PRMerger3** [comment]
@ptorr-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@ptorr-msft - resolved merge conflict... 

@TylerMSFT - can you review... or if there's someone else that should take this, can you add them? Thanks!

**Comment by TylerMSFT** [comment]
Adding @mmacy

**Comment by TylerMSFT** [comment]
Actually, looking at this more closely, it looks like a Windows UWP API. I'll ping the manager for that area to see who takes it.

**Comment by ptorr-msft** [comment]
It can be used by Desktop and UWP apps. There are very few "UWP" APIs per se.

**Review by alvinashcraft — APPROVED** [review]
So, I was doing some research to try and figure out if there are actually multiple lockers for a user, or if it's a single locker with different rules for credential access for UWP vs Desktop apps. None of the docs are specific about that, but they all refer to Credential Locker in the singular form.
Either way, I think that this updated description makes it clear that Desktop apps can access all of the current user's credentials from other apps. That should eliminate some confusion.

File: windows.security.credentials/passwordvault.md [hunk 1]
```diff
@@ -10,11 +10,14 @@ public class PasswordVault : Windows.Security.Credentials.IPasswordVault
 # Windows.Security.Credentials.PasswordVault
 
 ## -description
-Represents a Credential Locker of credentials. The contents of the locker are specific to the app or service. Apps and services don't have access to credentials associated with other apps or services.
+Represents a Credential Locker of credentials. Lockers are specific to a user.
+
+Apps running in an AppContainer (e.g. UWP apps) can only access the contents of their own locker (for the current user). Apps not running in an AppContainer (e.g. regular Desktop apps) can access all the user's lockers, including those of AppContainer apps.
```
Comment (ShannonLeavitt) [inline windows.security.credentials/passwordvault.md]:
```suggestion
Apps running in an AppContainer (for example, UWP apps) can only access the contents of their own locker (for the current user). Apps not running in an AppContainer (for example, regular Desktop apps) can access all the user's lockers, including those of AppContainer apps.
```
Fixing terminology

**Comment by PRMerger13** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
I'll merge per approval from @alvinashcraft above.

**Pull request merged**